File: API/Entities/Service.cs
namespace API.Entities;

public class Service
{
    public int Id { get; set; }
    public string ClientEmail { get; set; }
    public string ClientUsername { get; set; }
    public string Name { get; set; }
    public List<PictureUrlService> PictureUrls { get; set; } = new();
    public StatusOfService CurrentStatus { get; set; } = StatusOfService.NotStarted;
    public int Price { get; set; }
    public DateTime PlannedDateOfCompletion { get; set; } = DateTime.UtcNow;
    public string Description { get; set; }
    public List<Comment> Comments { get; set; } = new();
}

File: client/src/app/models/service.ts
export interface Service {
    id: number
    name: string
    pictureUrls: PictureUrl[]
    currentStatus: number
    price: number
    plannedDateOfCompletion: string
    description: string
    comments: Comment[]
  }
  
  export interface PictureUrl {
    id: number
    url: string
  }
  
  export interface Comment {
    id: number
    content: string
    user: string
    dateTime: string
  }
  